Source/Engine/Syntax/SearchTargetSyntax.cs

```diff
@@ -47,7 +40,7 @@ protected internal override Syntax Accept(SyntaxVisitor visitor)
     public class NamespaceSearchTargetSyntax : SearchTargetSyntax
     {
         public string Namespace { get; }
-        public ReadOnlyCollection<Syntax> PatternReferences { get; }
+        public ReadOnlyCollection<Syntax> PatternReferences { get; internal set; }
 
         internal NamespaceSearchTargetSyntax(string nameSpace, IList<Syntax> patternReferences)
```

Comment:
This constructor should not accept patternReferences, as it will always be empty at the moment of creation and is set during linking.